packages/linux/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.6.2"
+  changes:
+    - description: Fix hostfs values, version string
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/2201
```

Could you please split both changes in the changelog?
```suggestion
    - description: Fix hostfs values
      type: bugfix
      link: https://github.com/elastic/integrations/pull/2201
    - description: Support Kibana 8
      type: enhancement
      link: https://github.com/elastic/integrations/pull/2201
```